fdbclient/GlobalConfig.actor.cpp

```diff
@@ -155,7 +156,8 @@ void GlobalConfig::erase(KeyRangeRef range) {
 // function performs a one-time migration of data in these keys to the new
 // global configuration key space.
 ACTOR Future<Void> GlobalConfig::migrate(GlobalConfig* self) {
-	state Reference<ReadYourWritesTransaction> tr = makeReference<ReadYourWritesTransaction>(self->cx);
+	state Database db = Database(self->cx);
```

Comment:
I think this may still cause problems if `migrate` gets stuck (e.g. because the database is unavailable). It seems like this problem exists to some extent for other actors in `DatabaseContext`, but one thing those other actors do is to clear out the database reference on each retry. We'll need to do that when this integrates with the other changes to put this function in a retry loop.

The specific procedure for this is to not create the `db` as a state variable and to clear out the transaction reference on each retry. It may also be necessary to introduce a `delay(0)` as well, which is what is done in the `refreshTransaction` function.

I suppose we should also create an issue to deal with the case that this technique doesn't help if we block indefinitely on a read, etc.